### PR TITLE
News visual design

### PR DESCRIPTION
**Pull request by kobsonaut**

### CODE REVIEW

File: TOZ_iOS/GetNews.json [hunk 1]
```diff
@@ -1,42 +1,42 @@
...
-  "created": 1222333444555,
-  "lastModified": 1222333444555,
-  "published": 1483699792
-  }
-  ]
+[
+ {
+ "id": "c5296892-347f-4b2e-b1c6-6faff971f767",
+ "title": "WITAMY NOWEGO MIESZKAŃCA TOZ!",
+ "contents": "Piorun został odnaleziony przy piekarni na ulicy Krzemiennej, gdy samotnie błąkał (...) Czytaj dalej",
```
Comment (Gorczyk89) [inline TOZ_iOS/GetNews.json]: Why there is "(...) Czytaj dalej"? I think it should be other label for this.

File: TOZ_iOS/NewsResponseMapper.swift [hunk 2]
```diff
@@ -22,6 +22,7 @@ final class NewsResponseMapper: ArrayResponseMapper<NewsItem>, ResponseMapperPro
             var photoURL: URL? = nil
             if let photoUrl = photoUrl {
                 photoURL = BackendConfiguration.shared.baseURL.appendingPathComponent(photoUrl)
+//                photoURL = URL(string: photoUrl)
```
Comment (piotr-tobolski) [inline TOZ_iOS/NewsResponseMapper.swift]: If this is not needed, it can be removed

File: TOZ_iOS/GetNews.json (hunk 1 above)
Comment (piotr-tobolski) [inline TOZ_iOS/GetNews.json]: +1

File: TOZ_iOSTests/NewsResponseMapperTest.swift [hunk 3]
```diff
@@ -34,10 +34,11 @@ class NewsResponseMapperTest: XCTestCase {
...
-        XCTAssertEqual(firstNews?.photoUrl, BackendConfiguration.shared.baseURL.appendingPathComponent("storage/a9/2c/a92ccd6a-f51c-4ff0-8645-02adff409051.jpg"))
-        XCTAssertEqual(firstNews?.published, Date(timeIntervalSince1970: TimeInterval(1494352543)))
+        XCTAssertEqual(firstNews?.title, "WITAMY NOWEGO MIESZKAŃCA TOZ!")
+        XCTAssertEqual(firstNews?.contents, "Piorun został odnaleziony przy piekarni na ulicy Krzemiennej, gdy samotnie błąkał (...) Czytaj dalej")
+//        XCTAssertEqual(firstNews?.photoUrl, BackendConfiguration.shared.baseURL.appendingPathComponent("storage/a9/2c/a92ccd6a-f51c-4ff0-8645-02adff409051.jpg"))
+//        XCTAssertEqual(firstNews?.photoUrl, URL(string: "https://images.pexels.com/photos/47466/pexels-photo-47466.jpeg?w=1200&h=627&fit=crop&auto=compress&cs=tinysrgb"))
+        XCTAssertEqual(firstNews?.published, Date(timeIntervalSince1970: TimeInterval(1222333444555)))
     }
-
+
```
Comment (piotr-tobolski) [inline TOZ_iOSTests/NewsResponseMapperTest.swift]: Where is test for photoUrl? And why there are 2 commented lines? We shouldn't commit commented code.
If you are making changes to the unit test, you can also fix it - assertions should be in separate tests, e.g.
```
func testFirstNewsTitle() {
    XCTAssertEqual(firstNews?.title, "Do oddziału TOZ trafił nowy pies!")
}
```
And `let firstNews ` should be a property.

File: TOZ_iOS/GetNews.json [hunk 4]
```diff
@@ -1,42 +1,43 @@
...
-  "published": 1488736635
-  }
-  ]
+[
+ {
+ "id": "c5296892-347f-4b2e-b1c6-6faff971f767",
+ "title": "WITAMY NOWEGO MIESZKAŃCA TOZ!",
+ "contents": "Piorun został odnaleziony przy piekarni na ulicy Krzemiennej, gdy samotnie błąkał (...) Czytaj dalej",
+ "type": "RELEASED",
+ "photoUrl": "https://images.pexels.com/photos/47466/pexels-photo-47466.jpeg?w=1200&h=627&fit=crop&auto=compress&cs=tinysrgb",
```
Comment (piotr-tobolski) [inline TOZ_iOS/GetNews.json]: This is not aligned with real implementation, I suggest leaving relative paths there. If you want to have some photos mocked - we can modify our mock behavior to have host like e.g. imgur or placekitten and post relative paths to one of this services in the responses :-)

**Pull request merged**